regression-tests/graysynthetic/compare.py

```diff
@@ -3,8 +3,13 @@
 
 class CSVFile:
     def __init__(self, file):
-        self.cols = [col.strip() for col in next(file).split(',')]
+        
+        next_line = next(file,None)
```

Comment:
This looks to solve this particular issue, but I wonder what would happen later on if you gave it an empty file. Will it crash later on?
We should probably raise an exception somewhere if the `.csv` file is found to be empty.

Reply:
Thanks for the suggestion! I have added the code to check empty files in the beginning.

Reply:
Thanks for the suggestion! I have added the code to check empty files in the beginning.